action.yaml

```diff
@@ -162,7 +162,7 @@ runs:
         d$set_list(key, new_values)
       }
       desc_field_clear_if_empty <- function(d, key) {
-        if (d$has_fields(key) && d$get_field(key) == "") {
+        if (d$has_fields(key) && d$get_field(key, "") == "") {
```

Comment:
here we already have explicitely `d$has_fields(key)`  so the change seems to be redundant

Reply:
Without testing I didn't knew if an empty field would be reported as NULL or as "". I'll see what happens and fix accordingly but for the moment I removed it.

Reply:
`desc::description$new(file)`  return the field as "" (but warns users) so yes, this was redundant